process/block/baseProcess.go

```diff
@@ -1150,7 +1160,15 @@ func (bp *baseProcessor) getRootHashes(currHeader data.HeaderHandler, prevHeader
 	case state.UserAccountsState:
 		return currHeader.GetRootHash(), prevHeader.GetRootHash()
```

Comment:
why do we return here the same roothash two times ?

Reply:
it's not the same, it is root hashes from two different blocks, current & previous